src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,10 +24,14 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final DriveBase driveBase = new DriveBase(0, 1, 2, 3); // TODO: remove dummy port values
+  private final Outtake outtake = new Outtake(Constants.OUTTAKE_ID);
   private final Joystick primaryDriverJoystick =
       new Joystick(IOConstants.PRIMARY_DRIVER_JOYSTICK_PORT);
   private final Joystick secondaryDriverJoystick =
       new Joystick(IOConstants.SECONDARY_DRIVER_JOYSTICK_PORT);
+  private final JoystickButton secondaryDriverButton1 =
```

Comment:
Placeholder value should be labeled as a button constant in IOConstants

Rename "secondaryDriverButton1" to "outtakeButton"